fn/option.go

```diff
@@ -213,3 +216,12 @@ func (o Option[A]) UnsafeFromSome() A {
 	}
 	panic("Option was None()")
 }
+
+// String returns a string representation of the Option.
+func (o Option[A]) String() string {
+	if o.isSome {
+		return fmt.Sprintf("%v", o.some)
+	}
+
+	return "none"
```

Comment:
nit: should this be `[none]` or `<none>` to make it more explicit that the value isn't a string with the content `none` but a placeholder?

Reply:
Done!